google/data_source_storage_object_signed_url.go

```diff
@@ -68,7 +69,7 @@ func dataSourceGoogleSignedUrl() *schema.Resource {
 				Type:         schema.TypeString,
 				Optional:     true,
 				Default:      "GET",
-				ValidateFunc: validateHttpMethod,
+				ValidateFunc: validation.StringInSlice([]string{"GET", "HEAD", "PUT", "DELETE"}, true),
```

Comment:
Could add our own validation function that is 'isHTTPVerb' (although given not all the http verbs are here, maybe that would be more confusing?) What do you think?